hack/local-up-cluster.sh

```diff
@@ -1044,6 +1112,20 @@ if [[ "${KUBETEST_IN_DOCKER:-}" == "true" ]]; then
 
   # kubekins has a special directory for docker root
   DOCKER_ROOT="/docker-graph"
+
+  # to use docker installed containerd as kubelet container runtime
+  # we need to enable cri and install cni
+  # install cni for docker in docker
+  # containerd 1.4.12 installed by docker supports CNI version 0.4.0
+  CNI_VERSION=0.4.0
```

Comment:
this variable is duplicate and is not really needed, just hardcode 0.4.0 to the config file

>   "cniVersion": "${CNI_VERSION}",
to
>   "cniVersion": "0.4.0",

Reply:
actually there is a reason not directly hardcoded to config file, the function `install_cni` is intended to be used by directly by the user of the script, so they can set cni version as per there requirement

I would have hardcoded it to 1.0.0 but the containerd installed by docker in kubekins image does't support it

Reply:
this version is only the one that goes in the manifest conflist, is not needed

Reply:
you have CNI_VERSION at the beginning to represent the version of the binary

Reply:
but what i found is `CNI_VERSION` & `CNI_PLUGINS_VERSION` both are different 
`CNI_PLUGINS_VERSION` is the version of release plugins https://github.com/containernetworking/plugins/releases
`CNI_VERSION` is the spec version

Reply:
removed `CNI_VERSION`